components/tools/OmeroWeb/omeroweb/webgateway/static/webgateway/js/ome.viewport.js

```diff
@@ -1063,11 +1074,14 @@ jQuery._WeblitzViewport = function (container, server, options) {
     /* Channels (verbose as IE7 does not support Array.filter */
     var chs = [];
     var channels = this.loadedImg.channels;
+    console.log(channels);
     for (var i=0; i<channels.length; i++) {
       var ch = channels[i].active ? '' : '-';
       ch += parseInt(i, 10)+1;
       ch += '|' + channels[i].window.start + ':' + channels[i].window.end;
+      ch += channels[i].reverseIntensity ? 'r' : '-r';
```

Comment:
if you went with the proposal for ':' separation then it'd need to be

`ch += ':' + channels[i].reverseIntensity ? 'r' : '-r';`